MangaTaggerLib/utils.py

```diff
@@ -220,6 +224,8 @@ def load(cls):
         # Image Directory
         if settings['application']['image']['enabled']:
             cls.image = True
+            if settings['application']['image']['first']:

```

Comment:
This would raise exception if someone if not running docker.
```suggestion
            if settings['application']['image'].get('first'):
```